### PR TITLE
Expiration date

### PR DESCRIPTION
**Pull request by surabhishukla**

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -34,7 +34,9 @@ const App = () => {
       <LayoutWrapper>
         <Router />
       </LayoutWrapper>
-    </AppContext.Provider>
+      </AppContext.Provider>
+    //  <h1>Hello Geeks!!!</h1>
```
Comment (pinak-globallogic) [inline src/App.jsx:38]: remove

File: src/Pages/License/Generate/GenerateLicense.jsx [hunk 2]
```diff
@@ -24,21 +25,23 @@ export const CustomCardContent = styled(CardContent)(() => ({
...
+    case 0:
+      return <CustomerDetails />;
+    case 1:
+      return <ProductDetails />;
+    case 2:
+      return <FeatureDetails />;
+    case 3:
+      return <LimitationDetails />;
+    case 4:
+      return <GeneratrProductKeyDetails />;
```
Comment (pinak-globallogic) [inline src/Pages/License/Generate/GenerateLicense.jsx:37]: fix typo

File: src/Pages/License/Generate/LimitationDetails.jsx [hunk 3]
```diff
@@ -133,14 +133,14 @@ const LimitationDetails = () => {
                 <Grid item xs pr={2}>
                   <TextField
```
Comment (pinak-globallogic) [inline src/Pages/License/Generate/LimitationDetails.jsx:134]: Add ids to all UI elements

**Review by pinak-globallogic — CHANGES_REQUESTED** [review]

File: src/App.jsx (hunk 1 above)
Comment (surabhishukla) [reply src/App.jsx]: done

File: src/Pages/License/Generate/GenerateLicense.jsx (hunk 2 above)
Comment (surabhishukla) [reply src/Pages/License/Generate/GenerateLicense.jsx]: done

**Comment by pinak-globallogic** [comment]
@surabhishukla  Resolve conflicts and add package-lock.json to gitignore

**Pull request closed without merging**